### DISCUSSION TITLE
Add other html tag

### DISCUSSION THREAD

**Discussion by Ranggawahyu77** [awcodes/filament-tiptap-editor#477] [Q&A]
i want to add html tag like `<div>`, `<section>`. why when I save the tag it disappears, even though I have added this ``->output(TiptapOutput::Html)``

**Comment by awcodes** [comment]
Prosemirror, the underlying lib for Tiptap itself is very strict about what elements it allows, to do this you would need custom tiptap extensions to allow them.
